pkg/workloadmeta/testing/store.go

```diff
@@ -64,6 +64,43 @@ func (s *Store) ListContainersWithFilter(filter workloadmeta.ContainerFilterFunc
 	return res
 }
 
+// GetProcess implements Store#GetProcess.
```

Comment:
Should we add unit tests for these new 3 functions similarly to what has been done for `ListImages`?

https://github.com/DataDog/datadog-agent/commit/73ef8f4f77e320493d4bf8168cf08b88f765f829#diff-811b780603a941f7c6db8f69608edfe16baac25d62fe83a7e9503b3ce3433223R686

Reply:
Done as well!